sort_names.py

```diff
@@ -1,95 +1,188 @@
-import csv
+import csv  # work with .csv files
 import sys
 import time
+import urllib.request
+from collections import defaultdict
 from fileinput import filename
-from deepface import DeepFace
 
+# modules
+import filters
+import inputs
 
 filename = sys.argv[1]
 
 # ---- INPUT PARAMETERS ----
-limit_age = 25  # not younger than
-races = [
-    "black",
-    "white",
-    "asian",
-]  # enter necessary races. List of possible races - ['asian', 'indian', 'black', 'white', 'middle eastern', 'latino hispanic']
-forbidden_word = "NFT"  # a forbidden word in the current position of person
-black_list = "Indusi.csv"  # Filter by name. CSV file with two columns - First_name and Second_name
+LIMIT_AGE = 25  # not younger than
+# enter necessary races. List of possible races - ['asian', 'indian', 'black', 'white', 'middle eastern', 'latino hispanic']
+races = ["asian", "black", "white"]
+DEL_PEOPLE_WITHOUT_AVATAR = False  # past 'True' or 'False'
 
 
-def get_short_list(filename, first_names, last_names):
+def conditions(row, dict_races, first_names, last_names, positions):
+    """Check all conditions and if row is correct - return True, else - False.
+    At the same time returns a tuple with counters
+    """
+    if filters.name_filter(row[24], first_names):
+        if row[25] not in last_names:
+            if filters.position_filter(row[43], positions):
+                if (row[30] == "") and DEL_PEOPLE_WITHOUT_AVATAR:
+                    return False, (
```

Comment:
please make the second return parameter an object and return it.

Reply:
done, Thank you!

Reply:
Sorry, I thought that you see my comments after I re-request review